src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -355,26 +380,31 @@ def load_components(
     components = user_yaml.get("spec", "resources")["resources"].get("GenericItems")
     lq_name = local_queue or get_default_kueue_name(namespace)
     cluster_labels = labels
-    for component in components:
-        if "generictemplate" in component:
-            if (
-                "workload.codeflare.dev/appwrapper"
-                in component["generictemplate"]["metadata"]["labels"]
-            ):
-                del component["generictemplate"]["metadata"]["labels"][
+    if local_queue_exists(namespace, lq_name):
```

Comment:
small nit, can you change this to
```suggestion
    if not local_queue_exists(namespace, lq_name):
        raise ...
   // unindented default block  
   ...
```

Reply:
Updated.

Reply:
It looks the same to me.

Reply:
Apologies, I don't know what happened there. It is now updated.